reconcile/change_owners/change_log_tracking.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from reconcile.change_owners.bundle import (
+    NoOpFileDiffResolver,
+    QontractServerDiff,
+)
+from reconcile.change_owners.change_owners import fetch_change_type_processors
+from reconcile.change_owners.change_types import ChangeTypeContext
+from reconcile.change_owners.changes import aggregate_file_moves, parse_bundle_changes
+from reconcile.utils import gql
+from reconcile.utils.runtime.integration import NoParams, QontractReconcileIntegration
+from reconcile.utils.semver_helper import make_semver
+from reconcile.utils.state import init_state
+
+
+class ChangeLogIntegration(QontractReconcileIntegration[NoParams]):
+    def __init__(self) -> None:
+        super().__init__(NoParams())
+        self.qontract_integration = "change-log-tracking"
+        self.qontract_integration_version = make_semver(0, 1, 0)
+
+    @property
+    def name(self) -> str:
+        return self.qontract_integration
+
+    def run(self, dry_run: bool) -> None:
+        change_type_processors = [
+            ctp
+            for ctp in fetch_change_type_processors(
+                gql.get_api(), NoOpFileDiffResolver()
+            )
+            if ctp.labels and "change_log_tracking" in ctp.labels
+        ]
+        state = init_state(
+            integration=self.name,
+        )
```

Comment:
`init_state` requires `cleanup`, either use `defer` or `with`

Reply:
added in https://github.com/app-sre/qontract-reconcile/pull/4620/commits/18eca8143b38ef3fa18306b241168d2d5abc5a25